CharacterCreation/02_Classes.txt

```diff
@@ -325,6 +349,10 @@ Armor Proficiencies:
 * Light
 * Medium.
 
+Skill Proficiencies:
+* Vehicles - [Choose two]
+* Weapons - General
```

Comment:
Why does pilot seem to get a bonus to general weapons?

Reply:
I honestly have no idea; it was something Opossum wrote in.